### PR TITLE
Added --terminal and -t flag

### PR DESCRIPTION
**Pull request by tylerl0706**

### CODE REVIEW

File: resources/runners/qemu.sh [hunk 1]
```diff
@@ -1,3 +1,16 @@
+APPEND="root=/dev/sda ip=dhcp vga=0x344"
+while [ ! $# -eq 0 ]
+do
+    case "$1" in
+        --terminal | -t)
+            APPEND="$APPEND console=ttyAMA0,115200 console=ttyS0"
```
Comment (piranna) [inline resources/runners/qemu.sh]: What's `ttyAMA0` and why it's needed? I haven't seen it before...

File: resources/runners/qemu.sh [hunk 2]
```diff
@@ -1,3 +1,16 @@
...
+            QEMU="$QEMU -nographic"
+            echo "$APPEND $QEMU"
```
Comment (piranna) [inline resources/runners/qemu.sh]: Remove the `echo`.

File: resources/runners/qemu.sh (hunk 1 above)
Comment (tylerl0706) [inline resources/runners/qemu.sh]: I got it from here: http://stackoverflow.com/questions/19565116/redirect-qemu-window-output-to-terminal-running-qemu#comment42884295_19569340

Apparently some emulated machines use it. I can take it out if you think we should.

Comment (piranna) [inline resources/runners/qemu.sh]: Ok I see, in that case it's ok.

File: resources/runners/qemu.sh (hunk 2 above)
Comment (tylerl0706) [inline resources/runners/qemu.sh]: Removed :)

**Comment by piranna** [comment]
Cool :-) Could you be able to do the same for `iso.sh`? There's no need there for `--append`, after that I'll merge it :-)

**Comment by tylerl0706** [comment]
Did the same for `iso.sh` :)

**Comment by piranna** [comment]
And here it go! :-P

**Comment by tylerl0706** [comment]
Yay! Contribution! (I hope it works forever :neutral_face: )

**Pull request merged**